packages/neuron-wallet/tests/services/tx/transaction-generator.test.ts

```diff
@@ -380,13 +381,13 @@ describe('TransactionGenerator', () => {
       })
 
       describe('with full address', () => {
-        it(`only full address, 43 capacity`, async () => {
```

Comment:
This test case is meant to be unchanged, but it just fails anyway, I find that 43 CKBytes is not enough to send a transaction though, so it always throws out an error of capacity too small, I think it should be at least 61 ?